release-notes-1.0.1.md

```diff
@@ -0,0 +1,11 @@
+#### Version Number
+${version-number}
+
+#### New Features
+- None
+
+#### Patch Fixes Included
+- PostgreSQL JDBC driver version upgraded
```

Comment:
I made a comment on the commit, remaking it here in case you missed it. 
Should you have the Octane ticket number here?

Reply:
I have included the ticket number here.